index.rst

```diff
@@ -302,107 +305,153 @@ Is this case, we could have something like:
 
 .. _section-proposed-changes:
 
-Proposal for improvements
+Proposal for Improvements
 =========================
 
-The sections above describes the implementation of how CSC configuration is handled by the system, at the time of this writing.
+The sections above describe the implementation of how CSC configuration is handled by the system, at the time of this writing.
 During initial integration and tests we realized that the solution has some critical weaknesses that we need to address.
-This section describes some of the issues we found and propose changes to the system to improve the user experience and system reliability.
+This section describes some of the issues we found and proposes changes to the system to improve the user experience and system reliability.
 
-The following should be seen as an open floor for discussions and we expect developers and users to comment and provide feedback before we can start implementation.
+The following suggested implementation is still open for discussion and we encourage developers and users to comment and provide feedback before starting the process of implementation.
```

Comment:
"implementation process"?

Reply:
uhh, it's the same thing... but i'll change it.